transaction/core/src/tx_summary.rs

```diff
@@ -214,7 +214,8 @@ pub struct TxInSummary {
     #[prost(message, required, tag = "1")]
     pub pseudo_output_commitment: CompressedCommitment,
 
-    /// Whether there are input rules associated to this input
-    #[prost(bool, tag = "2")]
-    pub has_input_rules: bool,
+    /// If there are input rules associated to this input, the canonical digest
+    /// of these (per MCIP 52). If not, then this field is empty.
+    #[prost(bytes, tag = "2")]
+    pub input_rules_digest: Vec<u8>,
```

Comment:
:wrench: It looks like the current [MCIP-52](https://github.com/garbageslam/mcips/blob/tx_summary_digest/text/0052-tx-summary-digest.md) is still saying "flag".  While one might argue the presence or absence of this is a flag, I wonder if it would be better to be more specific in the MCIP
It could also be the iteration race of documenting and implementing and thus hasn't been updated yet.

Reply:
Yea im in the process of updating